components/splitview/index.css

```diff
@@ -93,78 +119,126 @@ governing permissions and limitations under the License.
       inset-inline-start: auto;
     }
   }
+
+  &.is-draggable {
+    &:hover,
+    &.is-hovered {
+      background-color: var(--highcontrast-splitview-handle-background-color-hover, var(--mod-splitview-handle-background-color-hover, var(--spectrum-splitview-handle-background-color-hover)));
+
+      .spectrum-SplitView-gripper {
+        border-color: var(--highcontrast-splitview-handle-background-color-hover, var(--mod-splitview-handle-background-color-hover, var(--spectrum-splitview-handle-background-color-hover)));
+        &:before {
+          background-color: var(--highcontrast-splitview-handle-background-color-hover, var(--mod-splitview-handle-background-color-hover, var(--spectrum-splitview-handle-background-color-hover)));
+        }
+      }
+    }
+
+    &:active,
+    &.is-active {
+      background-color: var(--highcontrast-splitview-handle-background-color-down, var(--mod-splitview-handle-background-color-down, var(--spectrum-splitview-handle-background-color-down)));
+
+      .spectrum-SplitView-gripper {
+        border-color: var(--highcontrast-splitview-handle-background-color-down, var(--mod-splitview-handle-background-color-down, var(--spectrum-splitview-handle-background-color-down)));
+        &:before {
+          background-color: var(--highcontrast-splitview-handle-background-color-down, var(--mod-splitview-handle-background-color-down, var(--spectrum-splitview-handle-background-color-down)));
+        }
+      }
+    }
+
+    &:focus {
+      outline: none;
+    }
+
+    &:focus-visible {
+      outline: none;
+      background-color: var(--highcontrast-splitview-handle-background-color-focus, var(--mod-splitview-handle-background-color-focus, var(--spectrum-splitview-handle-background-color-focus)));
```

Comment:
A separate story in Storybook could be nice if that's possible.

Reply:
I thought about that. The issue is that it's all divs so I can't trigger focus-visible without adding a class. I'm assuming that this focus is used in SWC? The CSS was set up like this but not sure how it works with no focusable elements

Reply:
Oh interesting ok the splitter in SWC has a tabindex of 0 so that's why this works there

Reply:
I'm going to have to do an interaction here to get the focus to show up without a class so this will have to wait until the Popover PR goes in... if it ever does